ui/app/initializers/enable-engines.js

```diff
@@ -0,0 +1,13 @@
+import config from '../config/environment';
+
+export function initialize(/* application */) {
+  // attach mount hooks to the environment config
+  // context will be the router DSL
+  config.addRootMounts = function() {
+    this.mount('replication');
+  };
+}
+
+export default {
+  initialize,
+};
```

Comment:
This initializer is key to the strategy we're going to adopt going forward. We'll be able to do logic / fetch info from the cluster, and then optionally enable engines we've abstracted from the current application. This will let us offer a more customized and streamlined experience for users (possibly even mixing up the route structure in the future).